src/controllers/__tests__/item.test.js

```diff
@@ -94,12 +94,40 @@ const noDescriptionItem = {
   type: 'Stationery',
 };
 
+const mockUser = {
+  username: 'chandler.bing',
+  firstName: 'Chandler',
+  lastName: 'Bing',
+  email: 'chandlerbing@gmail.com',
+  password: 'password1234',
+  passwordConfirm: 'password1234',
+  address: 'Central Perk, New York',
+  acceptTerms: true,
+};
+
 describe('Items Endpoints', () => {
   beforeAll(async () => {
     await clearDatabase();
+    const response = await request(server)
```

Comment:
Can you add a explainer comment above this explaining why you need the `authCookie` before all tests?

Reply:
Also since you've named the below response as `signInResponse` you can rename this to `signUpResponse`.